src/Telemetry/Events/recommended-widget.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Telemetry: Records events related to the Recommended Widget
+ *
+ * @package Parsely\Telemetry
+ * @since   3.10.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Telemetry;
+
+use WP_Widget;
+
+/**
+ * This is determined by the value passed to the `WP_Widget` constructor.
+ *
+ * @see https://github.com/Parsely/wp-parsely/blob/e9f1b8cd1a94743e068681a8106176d23857992d/src/class-parsely-recommended-widget.php#L28
+ */
+const PARSELY_RECOMMENDED_WIDGET_BASE_ID = 'parsely_recommended_widget';
```

Comment:
The value we pass in to WP_Widget is capitalized and should match here just to be safe. Alternatively, maybe it's better to lowercase whatever `id_base` value we receive just to be safe?